godot-kotlin/godot-extension-injection/src/native/kotlin/godot/extension/injection/InjectionHelperPropertyHandler.kt

```diff
@@ -0,0 +1,17 @@
+package godot.extension.injection
+
+import godot.core.NodePath
+import godot.core.Variant
+import kotlin.reflect.KMutableProperty0
+
+class InjectionHelperPropertyHandler(private val nodePath: KMutableProperty0<NodePath>) {
+    fun get(): Variant {
+        return Variant.wrap(
+            nodePath.get()
+        )
+    }
+
+    fun set(value: Variant) {
+        nodePath.set(value.unwrap())
```

Comment:
Same here. We already know this Variant contains a NodePath so you can directly write: 
`nodePath.set(value.asNodePath()`